msg/es_ES.utf8/utils.msg

```diff
@@ -453,7 +464,7 @@ $set 12 MSGCAT_UTIL_SET_LOADDB
 117 Carga fallida.\n
 118 La longitud máxima del nombre de la clase es %1$d bytes.\n
 119 Total de %1$8d objeto(s) verificado, %2$d objeto(s) falló. \n
-120 del archivo de datos no contiene '%%class' especificador o '--table' falta la opción \n
+120 del archivo de datos no contiene '%% class' especificador o '--table' falta la opción \n
```

Comment:
if you intended this change, I think this change will be applied to other msg files.

Reply:
No, I didn't. Thanks